src/pages/home/report/ReportActionsView.js

```diff
@@ -65,6 +65,10 @@ function ReportActionsView(props) {
 
     const reportScrollManager = useReportScrollManager();
 
+    const scrollToBottom = useCallback(() => {
+        reportScrollManager.scrollToBottom();
+    }, [reportScrollManager]);
+
```

Comment:
This isn't needed. This scrollToBottom function will be recreated on each re-render of the function.
Why? Because `reportScrollManager` isn't stable. It will be a new object on every re-render.
Thus the `useCallback` has to be applied inside the `reportScrollManager` module!

Reply:
I do think that this might cause some of the issues we are seeing at the minute